pkg/chartverifier/report.go

```diff
@@ -50,16 +46,16 @@ type ReportMetadata struct {
 }
 
 type ToolMetadata struct {
-	Version                string `json:"verifier-version" yaml:"verifier-version"`
-	ChartUri               string `json:"chart-uri" yaml:"chart-uri"`
-	Digest                 string `json:"digest" yaml:"digest"`
-	LastCertifiedTimestamp string `json:"lastCertifiedTimestamp" yaml:"lastCertifiedTimestamp"`
-
-	CertifiedOpenShiftVersions string `json:"certifiedOpenShiftVersions" yaml:"certifiedOpenShiftVersions"`
+	Version                    string `json:"verifier-version" yaml:"verifier-version"`
```

Comment:
I believe it is more common for fields to follow a `snakeCase` convention instead of `kebab-case`:

```suggestion
	Version                    string `json:"verifierVersion" yaml:"verifierVersion"`
```

Reply:
I updated the one I added profile-name to profileName but not the others because they are already in the reports and the submission flow may read them (at least chart-uri) e.g.: https://github.com/openshift-helm-charts/charts/blob/6ea45d9f90b8317e3b1853ec0f36fecc273a4b29/scripts/src/chartprreview/verify-report.sh#L98

Reply:
How would you solve this in the long run? Add another issue to perform the change on both systems atomically?